packages/esm-patient-chart-app/src/ui-components/patient-chart/patient-chart.component.tsx

```diff
@@ -16,48 +22,52 @@ interface PatientChartParams {
 const PatientChart: React.FC<RouteComponentProps<PatientChartParams>> = ({
   match
 }) => {
+  const layout = useLayoutType();
   const { patientUuid, view, subview } = match.params;
   const [loading, patient] = useCurrentPatient(patientUuid);
   const state = useMemo(() => {
     return { patient, patientUuid };
   }, [patient, patientUuid]);
 
   return (
-    <main
-      className="omrs-main-content"
-      style={{
-        display: "flex",
-        alignItems: "flex-start",
-        flexDirection: "column"
-      }}
-    >
-      {loading ? (
-        <Loader />
-      ) : (
-        <>
-          <ExtensionSlot extensionSlotName="breadcrumbs-slot" />
-          <aside className={styles.patientBanner}>
-            <ExtensionSlot
-              extensionSlotName="patient-header-slot"
-              state={state}
-            />
-            <ExtensionSlot
-              extensionSlotName="patient-info-slot"
-              state={state}
-            />
-          </aside>
-          <div className={styles.grid}>
-            <div className={styles.chartreview}>
-              <ChartReview {...state} view={view} subview={subview} />
-              <VisitDialog />
-            </div>
-            <div className={styles.workspace}>
-              <WorkspaceWrapper />
+    <>
+      <SideMenu />
+      <main
+        className="omrs-main-content"
+        style={{
+          display: "flex",
+          alignItems: "flex-start",
+          flexDirection: "column"
+        }}
+      >
+        {loading ? (
```

Comment:
Instead of `loading` it's better to use `isLoading`  for the variable name because it's a boolean and that's a common convention for boolean variable naming.

Reply:
Not really. I think you confuse this with Java or .NET.

It is common, but so is just using words like `loading`. There is no clear winner here.